### ISSUE TITLE
[Deno] Invalid JSON received from the Stripe API

### ISSUE THREAD

**Issue by thorwebdev** [stripe/stripe-node#1519]
### Describe the bug

`stripe.accounts.createLoginLink` throws error when executed in Deno environment.

### To Reproduce

1. Create index.ts file
2. Add code
3. Run via Deno
4. Get error

### Expected behavior

Request to execute successfully

### Code snippets

```JavaScript
// errors
const loginLink = await stripe.accounts.createLoginLink('acct_1LZWJGPPxRBiNZOx')

// works fine
const loginLink = await fetch(
  `https://api.stripe.com/v1/accounts/${'acct_1LZWJGPPxRBiNZOx'}/login_links`,
  {
    method: 'POST',
    headers: {
      Authorization: `Bearer ${Deno.env.get('STRIPE_API_KEY')}`,
    },
  }
).then((res) => res.json())
```

### OS

macOS 11.06.1

### Node version

Deno version: { deno: "1.20.3", v8: "10.0.139.6", typescript: "4.6.2" }

### Library version

https://esm.sh/stripe@9.6.0?target=deno&no-check

### API version

2020-08-27

### Additional context

Related [discussion](https://discord.com/channels/841573134531821608/841573134531821616/1011202044436152390) on StripeDev Discord

Potentially related to https://github.com/stripe/stripe-node/issues/954 ?

**Comment by dcr-stripe** [comment]
Thanks for the report @thorwebdev and sorry you're having issues! I'm able to reproduce this as well. Going to investigate and report back!

**Comment by dcr-stripe** [comment]
For some reason the response for these calls when issued from Deno is actually returning a 400 without the usual JSON content:

```
<html>
<head><title>400 Bad Request</title></head>
<body>
<center><h1>400 Bad Request</h1></center>
<hr><center>nginx</center>
</body>
</html>
```

This causes the error handler to throw. I don't have the same behavior in Node.

**Comment by dcr-stripe** [comment]
This seems to be caused specifically due to sending `Content-Length: 0` for this request. If I remove this header internally, then Deno can send the request successfully and gets an appropriate response. 🤔

**Comment by dcr-stripe** [comment]
This is reproducible with:

```
const loginLink = await fetch(
  `https://api.stripe.com/v1/accounts/<account id>/login_links`,
  {
    method: 'POST',
    headers: {
      Authorization: `Bearer ${Deno.env.get('STRIPE_API_KEY')}`,
      "Content-Length": 0,
    },
  }
).then((res) => res.json())
```

This throws an error.

**Comment by dcr-stripe** [comment]
The root cause issue here is Deno injects its own Content-Length header despite `stripe-node` providing a value for it, so the end request has this header twice and nginx returns a 400. I've filed https://github.com/denoland/deno/issues/15545 on the Deno side for this.

I'll investigate to see if we can fix this from the Stripe side in the interim.

**Comment by dcr-stripe** [comment]
Hi @thorwebdev - thanks for the patience here!

I've released the fix under stripe-node@10.4.0-beta.1 . It now works for me. Could you try it out with your code just to double check, and if all works, I'll release it to the main stable channel?

You can try this out using:

```
import Stripe from "https://esm.sh/stripe@10.4.0-beta.1?target=deno";
```

Thanks!

**Comment by thorwebdev** [comment]
@dcr-stripe I can confirm that `import Stripe from 'https://esm.sh/stripe@10.4.0-beta.1?target=deno&no-check'` is working for me both locally and when deployed to Supabase/Deno Deploy. Thanks so much for the fast turnaround!

**Comment by dcr-stripe** [comment]
This is now released as stripe@10.5.0 (https://github.com/stripe/stripe-node/releases/tag/v10.5.0) - thanks for the patience here!

**Comment by mehulmathur16** [comment]
Hi, I'm getting this error on deployment. It works great on local though. 

Node version : **14.17.0** 
Stripe version : **11.3.0** 
OS : **Windows 11**
Getting this error in API : `stripe.customers.create`
